build/common/scripts/setup-webhook.sh

```diff
@@ -0,0 +1,7 @@
+kubectl apply -f deploy/webhook.yaml
+while [ $? -ne 0 ]
+do 
+    sleep 3
+    echo "-----webhook install failed try again-----"
+    kubectl apply -f deploy/webhook.yaml
+done 
```

Comment:
What is this mark?

Reply:
Do you mean the red minus in GitHub? That means there's no newline at the end of the file. Technically that means the line is incomplete: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206, but it's just a nitpick.

Or do you mean, why did kubebuilder add this script (I'm guessing that's where it came from)? I don't think we need a whole script, just try to apply the yaml once and give up in the make target if it fails... I definitely don't like the potentially infinite loop here.